app/assets/stylesheets/components/navigation.scss

```diff
@@ -140,4 +140,25 @@
       }
     }
   }
+
+  &--item {
+    padding: $su-3;
+    text-align: center;
+    width: 100%;
+
+    &--current {
+      border-bottom: 4px solid var(--accent-brand);
+
+      .crayons-tabs__label {
```

Comment:
Any reason why this element exist instead of applying styles within to the parent element (`*--current`)? There's an extra span for that in the markup as well so we could probably drop it too.